src/System.Windows.Forms/src/System/Windows/Forms/ToolStripContainer.ToolStripContainerAccessibleObject.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using static Interop;
+
+namespace System.Windows.Forms
+{
+    public partial class ToolStripContainer
+    {
+        internal class ToolStripContainerAccessibleObject : ControlAccessibleObject
+        {
+            private readonly ToolStripContainer _owningToolStripContainer;
+
+            public ToolStripContainerAccessibleObject(ToolStripContainer owner) : base(owner)
+            {
+                _owningToolStripContainer = owner;
+            }
+
+            internal override object? GetPropertyValue(UiaCore.UIA propertyID)
+               => propertyID switch
+               {
+                   UiaCore.UIA.AutomationIdPropertyId
+                       => _owningToolStripContainer.Name,
```

Comment:
You don't need this field, please remove it above
```suggestion
                       => Owner.Name,
```